src/StakeRegistry.sol

```diff
@@ -203,15 +187,85 @@ contract StakeRegistry is VoteWeigherBase, StakeRegistryStorage {
         }
     }
 
-    /*******************************************************************************
-                    EXTERNAL FUNCTIONS - SERVICE MANAGER OWNER
-    *******************************************************************************/
+    /// @notice Create a new quorum and add the strategies and their associated weights to the quorum.
+    function createQuorum(
```

Comment:
this needs to push something to total stake history in order for the quorumExists condition to be true